elasticsearch.md

```diff
@@ -1233,6 +1239,15 @@ FORBIDDEN by default req={ ID:747832602--1038482600#1312150, TYP:SearchRequest,
 ```
 The above rule gets forbidden "by default". This means that no ACL block has matched the request, so ReadonlyREST's default policy of rejection takes effect. 
 
+#### Requests finished with `INDEX NOT FOUND`
+
+This is an example of such request:
+
+```
+INDEX NOT FOUND req={  ID:501806845-1996085406#74,  TYP:GetIndexRequest,  CGR:N/A,  USR:dev1 (attempted),  BRS:true,  KDX:null,  ACT:indices:admin/get,  OA:172.20.0.1/32,  XFF:null,  DA:172.20.0.2/32,  IDX:nonexistent*,  MET:GET,  PTH:/nonexistent*/_alias/,  CNT:<N/A>,  HDR:Accept-Encoding=gzip,deflate, Authorization=<OMITTED>, Connection=Keep-Alive, Host=localhost:32773, User-Agent=Apache-HttpClient/4.5.2 (Java/1.8.0_162), content-length=0,  HIS:[CONTAINER ADMIN-> RULES:[auth_key->false]], [dev1 indexes-> RULES:[auth_key->true, indices->false], RESOLVED:[user=dev1]], [dev2 aliases-> RULES:[auth_key->false]], [dev3 - no indices rule-> RULES:[auth_key->false]]  }
+```
+
+The state above is only possible for read-only ES requests for blocks containing `indices` rule. If all other rules within the block are matched, but only `indices` rule is mismatched, the final state of the block is forbidden due to index not found.  
```

Comment:
* define what a read-only request is for us (it's a ROR concept, not ES concept).
* articles